common/lib/xmodule/xmodule/video_module/bumper_utils.py

```diff
@@ -106,7 +108,7 @@ def get_bumper_sources(video):
     try:
         val_profiles = ["desktop_webm", "desktop_mp4"]
         val_video_urls = edxval_api.get_urls_for_profiles(video.bumper['edx_video_id'], val_profiles)
-        bumper_sources = filter(None, [val_video_urls[p] for p in val_profiles])
+        bumper_sources = [_f for _f in [val_video_urls[p] for p in val_profiles] if _f]
```

Comment:
The `_f` here seems to be a placeholder that `python-modernize` uses whenever this pattern is encountered; something like `url` would probably make this a little easier to read and understand.